Sources/PackageModel/Manifest.swift

```diff
@@ -180,14 +180,14 @@ public final class Manifest {
         }
 
         var requiredDependencies: Set<PackageIdentity> = []
-        for targetTriple in self.targetsRequired(for: products) {
-            for targetDependency in targetTriple.dependencies {
+        for linuxGNUTargetTriple in self.targetsRequired(for: products) {
```

Comment:
?

Reply:
automated Xcode renaming is bad at this, sorry...